qm2/scf.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import psi4
+
+np.set_printoptions(suppress=True, precision=4)
+
+
+def buildgeom:
+    mol = psi4.geometry("""
+    O
+    H 1 1.1
+    H 1 1.1 2 104
+    """)
+
+    # Build a molecule
+    mol.update_geometry()
+    mol.print_out()
+
+
+    e_conv = 1.e-6
+    d_conv = 1.e-6
+    nel = 5
+    damp_value = 0.20
+    damp_start = 5
+return mol 
+
+# Build a basis
+bas = psi4.core.BasisSet.build(mol,target="aug-cc-pvdz")
```

Comment:
Usually the convention is to capitalize vdz. It doesn't really matter, but what usually shows up in papers and stuff is aug-cc-pVDZ.